test/test_files/function/gds/basic.test

```diff
@@ -19,6 +19,25 @@ Alice|12
 Bob|12
 Carol|12
 Dan|12
+-STATEMENT CALL SHOW_PROJECTED_GRAPH() RETURN *;
+---- 2
+PKWO|table: person, table: organisation|table: knows, table: workAt
+PK|table: person|table: knows
+-STATEMENT CALL create_projected_graph('withFilter', {'person': {'filter': 'n.age > 35'}}, {'knows': {'filter': 'size(r.comments) > 5'}})
+---- ok
+-STATEMENT CALL SHOW_PROJECTED_GRAPH() RETURN *;
+---- 3
+PKWO|table: person, table: organisation|table: knows, table: workAt
+PK|table: person|table: knows
+withFilter|table: person predicate: n.age > 35|table: knows predicate: size(r.comments) > 5
```

Comment:
let's make this a bit prettier. Maybe always project as strucut where keys are table names and values are filter strings (empty if there is not filter.)